src/index.js

```diff
@@ -1,4 +1,9 @@
 // import './style.css';
 import display from './modules/display.js';
```

Comment:
- [ ] I suggest you uncomment the `./style.css` for the dist folder styling folder to work.